### ISSUE TITLE
Cannot download ubootwrite.py it says insufficient permissions

### ISSUE THREAD

**Issue by antonical** [riptidewave93/LEDE-MR33#15]
HI I am trying to flash my MR33 the instructions on google drive say to download ubootwrite.py I can see the folder and the file but it will not download as it says insufficient permissions to download. I could download the license and readme ok.

I would be grateful if you could amend the permissions so I could download and have a go at flashing my MR33.

Cheers
Tony

**Comment by howff** [comment]
I can confirm that the file ubootwrite.py is readable for me in Chrome and Edge, whether signed into google or not, and I have no special permissions!

**Comment by antonical** [comment]
Hi thanks. I can read it just not download it. I use GCloud all the time. The file will not download and save. Anyway moot now as I have recreated but it could be I am unlucky in the MR33 lottery!

I followed your instructions and can get a serial output stream but unfortunately for me I think I cannot do anything with it. The U-Boot version shows as 2017.07.RELEASE-g7ed34f31579 (Sep 29 2017 - 07:43:44 -0700).

I had to connect the 3.3.v and ground of my USB adaptor to get it to work then it worked perfectly.

If I understood correctly then there is no solution for this U-Boot version.

Shame really I have a few of these and wanted to build a really cheap but high quality mesh using openwrt for a local community project to get internet available throughout the community hall.

Any advice appreciated. 

Cheers
Tony
